api/src/test/java/org/openmrs/module/cohort/api/impl/CohortServiceImplTest.java

```diff
@@ -167,9 +167,11 @@ public void shouldVoidCohort() {
 		cohortMember.setPatient(patient);
 		cohortMember.setUuid("4834jk3-n34nm30-34nm34-348nl");
 		cohortMember.setCohort(cohortM);
+		cohortM.addMemberships(cohortMember);
```

Comment:
It would be better to write a separate test for this. The test here isn't really valid because ultimately we mock the `createOrUpdate()` call...

Reply:
@ibacher This test is to check whether the child `cohortMember` is voided when the parent cohort is deleted. So I added the `cohortMember` to the `cohort`.